pkg/api/routes.go

```diff
@@ -149,5 +155,11 @@ func (h *Handler) GetRoutes() []Route {
 			"/log",
 			h.SetLogLevel,
 		},
+		{
+			"GetMetrics",
+			"GET",
+			"/metrics",
```

Comment:
Would you estimate there is any performance impact by collecting the metrics? I would hope not, but it would be good to know that we're not impacting the cluster by asking for its metrics. For now maybe we just keep an eye on the logging that says how long the http request took